README.md

```diff
@@ -377,6 +377,29 @@ okta:
     rateLimit:
       maxRetries: 4
 ```
+
+When you use OAuth 2.0, the full YAML configuration would look like:
+
+```yaml
+okta:
+  client:
+    connectionTimeout: 30 # seconds
+    orgUrl: "https://{yourOktaDomain}" # i.e. https://dev-123456.oktapreview.com
+    proxy:
+      port: null
+      host: null
+      username: null
+      password: null
+    authorizationMode: "PrivateKey"
+    clientId: "{clientId}"
+    scopes:
+      - okta.clients.read
```

Comment:
From what I know, PHP SDK config accepts this in a single line
`scopes: okta.clients.read okta.clients.write`
Does PHP also support this format of having each scope in new line? @bretterer 

Can we standardize the format across SDKs?

Reply:
@vijetmahabaleshwar-okta the spec at https://github.com/okta/oss-technical-designs/pull/20/files#diff-576923ebcba2a4823ed2cd6f3a9412ed notes the scopes defined in new lines. Therefore, I implemented it the same way :)

Reply:
@arvindkrishnakumar-okta we should make sure we have a test to validate loading the scopes from the ENV var

Reply:
@bdemers yes, there exists one already :)